### PR TITLE
update to _replaceRdd with tests

### PR DESCRIPTION
**Pull request by akmorrow13**

### CODE REVIEW

File: adam-python/bdgenomics/adam/rdd.py [hunk 1]
```diff
@@ -692,6 +692,11 @@ def _inferConversionFn(self, destClass):
 class FeatureRDD(GenomicDataset):
 
 
+    def _replaceRdd(self, newRdd):
```
Comment (heuermh) [inline adam-python/bdgenomics/adam/rdd.py]: Is this only necessary in `class FeatureRDD`?

**Comment by akmorrow13** [comment]
@heuermh if it is not present transform will fail (_replaceRdd is not a function)

File: adam-python/bdgenomics/adam/rdd.py (hunk 1 above)
Comment (akmorrow13) [reply adam-python/bdgenomics/adam/rdd.py]: @heuermh if it is not present transform will fail (_replaceRdd is not a function)

Comment (akmorrow13) [reply adam-python/bdgenomics/adam/rdd.py]: https://github.com/akmorrow13/adam/blob/eec00a545dca3e4cd4e258023c99a1afa8a8974e/adam-python/bdgenomics/adam/rdd.py#L104 is defined in GenomicRDD, calling _replaceRDD, which FeatureRDD does not have

Comment (heuermh) [reply adam-python/bdgenomics/adam/rdd.py]: Thanks!

**Comment by heuermh** [comment]
Thank you, @akmorrow13

**Pull request merged**